plugins/vli/vli-usbhub-lenovo.quirk

```diff
@@ -167,7 +167,13 @@ CounterpartGuid = USB\VID_17EF&PID_3095
 [DeviceInstanceId=USB\VID_17EF&PID_3095]
 Plugin = vli
 GType = FuVliUsbhubDevice
-Flags = usb2,has-shared-spi-pd,attach-with-gpiob
+Flags = usb2
+ParentGuid = USB\VID_17EF&PID_3094
+[DeviceInstanceId=USB\VID_17EF&PID_3097]
+Plugin = vli
+GType = FuVliUsbhubDevice
+Flags = usb2
+ParentGuid = USB\VID_17EF&PID_3095
```

Comment:
so this is a 2nd tier device? Why remove the other flags?

Reply:
VL211 is a 2nd tier device that only has a U2 port and does not share spi.

Reply:
But the VL817-Q7 still shares an SPI with the VL103-Q4, no?

Reply:
Oh, I understand your question now. Yes, VL817-Q7 usb2 can have the same flags "has-shared-spi-pd,attach-with-gpiob" as usb3. I didn't explain that our convention for VL817-Q7 usb2 is debugging only. As in, the end client will not update FW through VL817-Q7 usb2. The design choice is up to you, either way works!